gluten-core/src/main/scala/org/apache/spark/shuffle/sort/ColumnarShuffleManager.scala

```diff
@@ -80,34 +64,14 @@ class ColumnarShuffleManager(conf: SparkConf) extends ShuffleManager with Loggin
     mapTaskIds.synchronized {
       mapTaskIds.add(context.taskAttemptId())
     }
```

Comment:
Shall we move this code in ` case columnarShuffleHandle: ColumnarShuffleHandle` ?

Reply:
Moved to `case columnarShuffleHandle: ColumnarShuffleHandle`